core/src/main/java/com/netflix/conductor/core/execution/mapper/LambdaTaskMapper.java

```diff
@@ -68,7 +66,7 @@ public TaskType getTaskType() {
                 taskDefinition);
 
         Task lambdaTask = new Task();
-        lambdaTask.setTaskType(TASK_TYPE_LAMBDA);
+        lambdaTask.setTaskType(TaskType.TASK_TYPE_LAMBDA);
```

Comment:
Please mark this task mapper as deprecated

Reply:
Please also mark the DecisionTaskMapper as deprecated

Reply:
Done